Source/AppAuthTV/OIDTVServiceConfiguration.m

```diff
@@ -44,6 +45,15 @@ - (instancetype)initWithAuthorizationEndpoint:(NSURL *)authorizationEndpoint
                                 tokenEndpoint:(NSURL *)tokenEndpoint
     OID_UNAVAILABLE_USE_INITIALIZER(@selector(initWithTVAuthorizationEndpoint:tokenEndpoint:))
 
+- (instancetype)initWithDiscoveryDocument:(OIDServiceDiscovery *)discoveryDocument {
+  self = [super initWithDiscoveryDocument:discoveryDocument];
+
+  if (self) {
+    _TVAuthorizationEndpoint = [discoveryDocument.deviceAuthorizationEndpoint copy];
```

Comment:
How should we handle nil here? What happens if the user attempts to use the OIDTVServiceConfiguration with a nil _TVAuthorizationEndpoint?

Reply:
I tested, and it gives an unsupported URL error. I tried to catch this with this check in the service https://github.com/openid/AppAuth-iOS/pull/572/files#diff-0ba4897b100bdc309d5ed2a146f3a331R68

Reply:
OK, I can get a `OIDServiceDiscovery` object not using your method.

Can you add a `NSLog` here to alert developers to the condition?